hw/dv/tools/dvsim/sim.mk

```diff
@@ -94,13 +92,14 @@ ifneq (${sw_images},)
 				bazel_cmd="./bazelisk.sh"; \
 			else \
 				echo "Building \"$${bazel_label}\" on air-gapped machine."; \
-				bazel_opts+="${sw_build_opts} --distdir=$${BAZEL_DISTDIR} --repository_cache=$${BAZEL_CACHE}"; \
+				bazel_opts+=" --distdir=$${BAZEL_DISTDIR} --repository_cache=$${BAZEL_CACHE}"; \
 				bazel_cmd="bazel"; \
 			fi; \
+			echo "Building with command: $${bazel_cmd} build $${bazel_opts} $${bazel_label}"; \
 			$${bazel_cmd} build $${bazel_opts} $${bazel_label}; \
 			find -L $$($${bazel_cmd} info output_path)/ \
-				-type f -name "$$(basename $${image})*" | \
-				xargs -I % cp -f % $${target_dir}; \
+				-type f -name "$${bazel_target}*" | \
+				xargs -I % sh -c 'cp -f % $${run_dir}/$$(basename %)'; \
```

Comment:
Why not `ln -s`?

Reply:
We had a lot of issues with this on partner systems (see #13087) so I do not want to change this and risk disturbing a working setup for our partners.

Reply:
Ok, SG.